test/ctia/events_test.clj

```diff
@@ -75,3 +75,53 @@
     (is (= 1 (-> (<!! output) :entity :data)))
     (is (= 2 (-> (<!! output) :entity :data)))
     (is (nil? (poll! output)))))
+
+(deftest to-update-event-test
```

Comment:
`ctia.events` implements an event channel, and `ctia.events-test` is dedicated to test it.
IMHO this test belongs to an ns dedicated to test `ctia.entity.event.obj-to-event` functions which is lacking.
Could you move this test please?

Reply:
Great catch, moved to `ctia.entity.event.obj-to-event-test`.